internal/server/warning.go

```diff
@@ -79,6 +82,15 @@ func Warning(client WarningClient, tmpl template.Template) Handler {
 				return err
 			} else {
 				data.Success = true
+
+				SetFlash(w, FlashNotification{
+					Title: "Warning created",
+				})
+				for _, lpa := range data.Cases {
+					if lpa.CaseType == "DIGITAL_LPA" && slices.Contains(caseIDs, lpa.ID) {
+						return RedirectError(fmt.Sprintf("/lpa/%s", lpa.UID))
+					}
```

Comment:
Same here with flash messages:
```suggestion
				for _, lpa := range data.Cases {
					if lpa.CaseType == "DIGITAL_LPA" && slices.Contains(caseIDs, lpa.ID) {
						SetFlash(w, FlashNotification{
							Title: "Warning created",
						})
						return RedirectError(fmt.Sprintf("/lpa/%s", lpa.UID))
					}
```